region/maridia/inner-pink/Botwoon Hallway.json

```diff
@@ -217,6 +217,71 @@
         "With no jump assists, use a frozen Mochtroid as a platform to get to the ledge above the door."
       ]
     },
+    {
+      "link": [1, 2],
+      "name": "Temporary Blue Chain",
+      "entranceCondition": {
+        "comeInWithTemporaryBlue": {
+          "direction": "right"
+        },
+        "comesThroughToilet": "any"
+      },
+      "requires": [
+        "canChainTemporaryBlue",
+        {"or": [
+          {"and": [
+            "canSpringBallBounce",
+            {"or": [
+              {"ammo": {"type": "PowerBomb", "count": 2}},
+              "canLongChainTemporaryBlue"
+            ]}
+          ]},
+          {"and": [
+            "canLongChainTemporaryBlue",
+            "canStationaryLateralMidAirMorph",
+            "canBeVeryPatient"
+          ]}
+        ]}
+      ],
+      "note": [
+        "Chain temporary blue across the room in order to break the Speed blocks.",
+        "If available, bouncing using Spring Ball can help speed this up significantly.",
+        "If two Power Bombs are also available, they can be used to break the shot blocks along the way without needing to stop bouncing."
```

Comment:
I keep hitting the wall, when it opens up in the middle.  So I unmorphed there to avoid that.
Jumping over the wall at the start is a trickyJump - harder than very hard but within `canChainTemporaryBlue`

Reply:
The slow path jump morphs are can4HighMidairMorph too.

Reply:
Added those can4HighMidairMorph reqs. For the Power Bomb version, I found I could consistently avoid hitting the wall by setting up the bounce in a certain way; for the right-to-left version I was doing a stationary lateral mid-air morph at a certain position, whereas for left-to-right I was starting with a regular lateral mid-air morph.